src/components/conversation.js

```diff
@@ -91,7 +93,7 @@ class Conversation extends Component {
     for (const key in filesObj) {
       if (filesObj.hasOwnProperty(key)) {
         // Base64 encoding will inflate size to roughly 4/3 of original
-        if ((filesObj[key].size / 3 * 4) > imageMaxSize) {
+        if ((filesObj[key].size / 3) * 4 > imageMaxSize) {
```

Comment:
Check out @franckc 's comment here and explanation right above that: https://github.com/OriginProtocol/origin-dapp/pull/597#issuecomment-424972125
I think you can allow the actual file size to be up to 2MB because it gets decoded in origin-js layer before it gets submitted to IPFS - no?